### DISCUSSION TITLE
Does LSP mode have a way to uninstall servers?

### DISCUSSION THREAD

**Discussion by wildwestrom** [emacs-lsp/lsp-mode#3431] [Q&A] [answered]
`lsp-mode` has a command called `lsp-install-server`, but I don't see `lsp-uninstall-server` or `lsp-remove-server` anywhere.

Was this just never implemented? 
Do the maintainers just think this is unnecessary?
What implementation details must be accounted for to successfully patch this?

I just don't like seeing servers that I don't want to use, listed as installed. I know I can copy paste the npm command used to install and just make it uninstall in the same way, but that still doesn't change the list of installed servers.

**Comment by jcs090218** [comment]
I would say it's unnecessary, but it's indeed a nice feature to have!

**Comment by jcs090218** [comment, marked as answer]
It's implemented in https://github.com/emacs-lsp/lsp-mode/pull/3850.
